openedx/core/djangoapps/content_libraries/tests/test_views_lti.py

```diff
@@ -16,12 +17,23 @@ class LtiToolJwksViewTest(TestCase):
     Test JWKS view.
     """
 
+    def test_when_lti_disabled_return_404(self):
+        """
+        Given LTI toggle is disabled
+        When JWKS requested
+        Then return 404
+        """
+        response = self.client.get(URL_LIB_LTI_JWKS)
+        self.assertEqual(response.status_code, 404)
+
     def test_when_no_keys_then_return_empty(self):
         """
         Given no LTI tool in the database.
         When JWKS requested.
         Then return empty
         """
+        settings.FEATURES['ENABLE_CONTENT_LIBRARIES'] = True
+        settings.FEATURES['ENABLE_CONTENT_LIBRARIES_LTI_TOOL'] = True
```

Comment:
Probably better to use [`@override_settings`](https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.override_settings) - search this codebase for examples of using it to change `FEATURES` for a specific test.